tests/e2e/run-ui-e2e.sh

```diff
@@ -22,6 +22,11 @@ test_ui_e2e() {
     remove_existing_stackrox_resources
     setup_default_TLS_certs
 
+    [[ "$tag" =~ $RELEASE_RC_TAG_BASH_REGEX ]] || is_nightly_run || {
+        info "Disabling postgres as this is a release, RC or nightly build"
```

Comment:
Not familiar with this. but why limit Postgres testing  to dev builds?

Reply:
See this thread. Request from UI team: https://srox.slack.com/archives/C3W3U0RGW/p1664977840406369?thread_ts=1664927011.086369&cid=C3W3U0RGW